### ISSUE TITLE
Custom data process

### ISSUE THREAD

**Issue by jingwu2121** [zjy526223908/DreamEditor#11]
Hi, thank you so much for your great work! We are quite interested in it and hope to reproduce the result on our data. May I know how I can have e.g. `cameras_large.npz` and `horse.ply`? I would really appreciate it if you help me!

**Comment by 271806** [comment]
Did you found any information?
We really need a clearer instruction!

**Comment by jingwu2121** [comment]
No, unfortunately, no info here, didn't manage to do custom dataset process :(

**Comment by 271806** [comment]
@jingwu2121 Thanks for the reply, may I ask that which platform/OS you're running this repo? I started yesterday and I'm still stuck in running the the 'step2' script with many errors, on Windows 11. I think some dependencies might not fully support win11..

**Comment by jingwu2121** [comment]
I am running on Ubuntu 22.04.3 LTS. I am good to run step2, but keeping stuck on step3 with some weird Linux or CUDA errors 😢
